pom.xml

```diff
@@ -1017,6 +1018,13 @@ Copyright 2014-2016 CyberVision, Inc.
                 <artifactId>postgresql</artifactId>
                 <version>${pgsql.driver.ver}</version>
             </dependency>
+
+            <dependency>
```

Comment:
This dependency should be included only if profile is enabled. See this [link](http://stackoverflow.com/questions/166895/different-dependencies-for-different-build-profiles-in-maven) for details